### PR TITLE
Add aat + perftest s2s config

### PR DESCRIPTION
**Pull request by luigibk**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/perftest/common/rpe/rpe-service-auth-provider.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+spec:
+  releaseName: rpe-service-auth-provider
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: rpe-service-auth-provider
+    version: 0.2.0
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
```
Comment (adusumillipraveen) [inline k8s/perftest/common/rpe/rpe-service-auth-provider.yaml]:
```suggestion
      aadIdentityName: rpe
```

File: k8s/aat/common/rpe/rpe-service-auth-provider.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+spec:
+  releaseName: rpe-service-auth-provider
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: rpe-service-auth-provider
+    version: 0.2.0
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
```
Comment (adusumillipraveen) [inline k8s/aat/common/rpe/rpe-service-auth-provider.yaml]:
```suggestion
      aadIdentityName: rpe
```

File: k8s/perftest/common/rpe/rpe-service-auth-provider.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      applicationInsightsInstrumentKey: "ed3c45ae-01e1-4263-b57f-178a1ba004be"
+      image: hmctspublic.azurecr.io/hmcts/rpe-service-auth-provider:prod-9b79e227
+      ingressHost: rpe-service-auth-provider.service.core-compute-aat.internal
```
Comment (adusumillipraveen) [inline k8s/perftest/common/rpe/rpe-service-auth-provider.yaml]: this looks wrong

File: k8s/perftest/common/rpe/rpe-service-auth-provider.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  releaseName: rpe-service-auth-provider
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: rpe-service-auth-provider
+    version: 0.2.0
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "ed3c45ae-01e1-4263-b57f-178a1ba004be"
```
Comment (adusumillipraveen) [inline k8s/perftest/common/rpe/rpe-service-auth-provider.yaml]: may be it should be different for perftest?

File: k8s/perftest/common/rpe/rpe-service-auth-provider.yaml (hunk 3 above)
Comment (luigibk) [reply k8s/perftest/common/rpe/rpe-service-auth-provider.yaml]: Please do not leave comments like the above :-)
Rest instead!

Comment (adusumillipraveen) [reply k8s/perftest/common/rpe/rpe-service-auth-provider.yaml]: sure..

File: k8s/aat/common/rpe/rpe-service-auth-provider.yaml [hunk 5]
```diff
@@ -14,9 +14,10 @@ spec:
     name: rpe-service-auth-provider
     version: 0.2.0
   values:
+    rollback:
+      enable: true
     java:
       replicas: 2
-      useInterpodAntiAffinity: true
```
Comment (adusumillipraveen) [inline k8s/aat/common/rpe/rpe-service-auth-provider.yaml]: We still need this.. My bad i added it incorrectly before.

**Pull request merged**